plugin/Controller/ContentFilter/RenderHtmlHelperUsers.php

```diff
@@ -36,14 +36,18 @@ class RenderHtmlHelperUsers
      */
     public static function renderHtmlHelperUserIfEmptyId(string $type, string $documentationLink, string $linkDetail = null, array $pDataDetail): string
     {
+        /* translators: %s: type (e.g., estate, address) */
         $title       = sprintf(__("%s list documentation", 'onoffice-for-wp-websites'), ucfirst($type));
         $linkDetail  = !empty($pDataDetail) ? $linkDetail : '<a href=' . esc_url($documentationLink) . '>' . esc_html($title) . '</a>';
-        $description = sprintf(__("The plugin couldn't find any %s. Please make sure that you have published some %s, as described in the %s", 'onoffice-for-wp-websites'), $type, $type, $linkDetail);
+        /* translators: 1: type (e.g., estate, address), 2: type (e.g., estate, address), 3: link to documentation */
+        $description = sprintf(__("The plugin couldn't find any %1\$s. Please make sure that you have published some %2\$s, as described in the %3\$s", 'onoffice-for-wp-websites'), $type, $type, $linkDetail);
```

Comment:
Wenn du schon unbedingt das `$ Format` nutzen willst, dann verwende es auch wofür es gemacht wurde :P 
```suggestion
        /* translators: 1: type (e.g., estate, address), 2: type (e.g., estate, address), 3: link to documentation */
        $description = sprintf(__("The plugin couldn't find any %1\$s. Please make sure that you have published some %1\$s, as described in the %2\$s", 'onoffice-for-wp-websites'), $type, $linkDetail);
```
```